dev/AppLifecycle/ProtocolActivatedEventArgs.h

```diff
@@ -10,21 +10,21 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
     using namespace winrt::Windows::Foundation::Collections;
     using namespace winrt::Windows::ApplicationModel::Activation;
 
-    static PCWSTR c_protocolContractId = L"Windows.Protocol";
+    constexpr PCWSTR c_protocolContractId = L"Windows.Protocol";
 
-    class ProtocolActivatedEventArgs : public winrt::implements<ProtocolActivatedEventArgs, ActivatedEventArgsBase, IProtocolActivatedEventArgs,
+    class ProtocolActivatedEventArgs : public winrt::implements<ProtocolActivatedEventArgs, IProtocolActivatedEventArgs, ActivatedEventArgsBase,
         IInternalValueMarshalable>
     {
     public:
-        ProtocolActivatedEventArgs(const std::wstring& uri) : m_uri(winrt::Windows::Foundation::Uri(uri))
+        ProtocolActivatedEventArgs(const std::wstring& uri) : m_uri(winrt::Windows::Foundation::Uri(std::move(uri)))
```

Comment:
>wstring [](http://example.com/codeflow?start=46&length=7)

This seems silly. We start with an `hstring` returned from `GetFirstValueByName()`, convert it to a `wstring` so we can pass it to the `ProtocolActivatedEventArgs` constructor, and that constructor takes the `wstring` and converts it back to an `hstring` so we can construct a `Uri`from it.

Why not just leave it as an `hstring` all the way to the bottom?